apps/admin-gui/src/app/vos/pages/group-detail-page/group-attributes/group-attributes.component.ts

```diff
@@ -47,6 +55,8 @@ export class GroupAttributesComponent implements OnInit {
   selection = new SelectionModel<Attribute>(true, []);
   attributes: Attribute[] = [];
   groupId: number;
+  members: RichMember[] = [];
+  resources: Resource[] = [];
```

Comment:
dunno why you added this, it doesnt seems like you are using it somewhere
if thats the case please delete also unused things from constructor and import